src/System.Private.CoreLib/src/System/Runtime/RuntimeImports.cs

```diff
@@ -212,6 +212,9 @@ internal static void RhWaitForPendingFinalizers(bool allowReentrantWait)
                                                     out UIntPtr lastRecordedHeapSize,
                                                     out UIntPtr lastRecordedFragmentation);
 
+        [DllImport(RuntimeLibrary, ExactSpelling = true)]
+        internal static unsafe extern void RhAllocateUninitializedArray(IntPtr pEEType, UIntPtr totalSize, uint cElements, void* v);
```

Comment:
Better name for the last argument? E.g. `pResult` ?

Reply:
Oops, that's what you get when you carelessly let roslyn generate that parameter for you